spec/controllers/api/encryptables_controller_spec.rb

```diff
@@ -776,10 +775,38 @@
       get :show, params: { id: encryptable_file.id }, xhr: true
     end
 
-    it 'displays transferred encryptable and dont download it' do
+    it 'download transferred encryptable new encryption algorithm' do
+      login_as(:bob)
+
+      encryptable_file = prepare_transferred_encryptable(Crypto::Symmetric::Aes256iv)
+
+      login_as(:alice)
+
+      expect(controller).to receive(:send_file).exactly(:once)
+
+      get :show, params: { id: encryptable_file.id }, xhr: true
+    end
+
+    it 'displays transferred encryptable and dont download it old encryption algorithm' do
```

Comment:
please make this text a bit clearer.

you could also group these inside a describe block